### PR TITLE
Combined language file update 2

### PR DESCRIPTION
**Pull request by Dillon-Brown**
## Description

Combined PR for the removal of unused strings. Superseding #3349

## Motivation and Context

Second part of combined language files update as described by @horus68 #3346 

- Issue reference:  #2737 
- Issue reference:  #2736 
- Issue reference:  #2735 
- Issue reference:  #2724 
- Issue reference:  #2723 
- Issue reference:  #2722 
- Issue reference:  #2702 
- Issue reference:  #2699 
- Issue reference: #2687
- Issue reference: #2681
- Issue reference: #2681
- Issue reference: #2680
- Issue reference: #2679
- Issue reference: #2676
- Issue reference: #2668
- Issue reference: #2669
- Issue reference: #2666
- Issue reference: #2665
- Issue reference: #2658
- Issue reference: #2656
- Issue reference: #2653
- Issue reference: #2650
- Issue reference: #2647
- Issue reference: #2627
- Issue reference: #2615
- Issue reference: #2614
- Issue reference: #2583 
- Issue reference: #2580
- Issue reference: #2576
- Issue reference: #2568
- Issue reference: #2561
- Issue reference: #2558
- Issue reference: #2552
- Issue reference: #2535
- Issue reference: #2534
- Issue reference: #2529
- Issue reference: #2525
- Issue reference: #2524
- Issue reference: #2521
- Issue reference: #2518
- Issue reference: #2513
- Issue reference: #2508
- Issue reference: #2501
- Issue reference: #2491
- Issue reference: #2485
- Issue reference: #2477
- Issue reference: #2468
- Issue reference: #2468
- Issue reference: #2467
- Issue reference: #2467
- Issue reference: #2705
- Issue reference: #2708
- Issue reference: #3396
- Issue reference: #3416 
- Issue reference: #3489 
- Issue reference: #2749 
- Issue reference: #3430
- Issue reference: #2972
- Issue reference: #2703 

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: modules/Schedulers/language/en_us.lang.php [hunk 1]
```diff
@@ -139,16 +139,12 @@
...
-    'ERR_DELETE_RECORD' => 'You must specify a record number to delete the schedule.',
     'ERR_CRON_SYNTAX' => 'Invalid Cron syntax',
-    'NTC_DELETE_CONFIRMATION' => 'Are you sure you want to delete this record?',
-    'NTC_STATUS' => 'Set status to Inactive to remove this schedule from the Scheduler dropdown lists',
     'NTC_LIST_ORDER' => 'Set the order this schedule will appear in the Scheduler dropdown lists',
     'LBL_CRON_INSTRUCTIONS_WINDOWS' => 'To Setup Windows Scheduler',
     'LBL_CRON_INSTRUCTIONS_LINUX' => 'To Setup Crontab',
     'LBL_CRON_LINUX_DESC1' => 'In order to run SuiteCRM Schedulers, edit your web server user\'s crontab file with this command: ',
     'LBL_CRON_LINUX_DESC2' => '... and add the following line to the crontab file: ',
-    'LBL_CRON_LINUX_DESC3' => 'You should do this only after the installation is concluded.',
```
Comment (horus68) [inline modules/Schedulers/language/en_us.lang.php]: @Dillon-Brown the string `'LBL_CRON_LINUX_DESC3' ` is still in use here:
https://github.com/salesagility/SuiteCRM/blob/hotfix/install/ready.php#L292

**Comment by horus68** [comment]
@Dillon-Brown made some reviews on duplicated strings, mostly the double-spaces effect. 
@samus-aran its time for this PR to be considered a priority to be merged. It has more then 1000 strings to be removed from translations and this is an extra load of work for translators.
I've been able to hidden all those strings from Crowdin translators.
Because this is not merged, to keep Crowdin in sync for next versions will become a task to big to be handled: I have to monitor the Hotfix releases and the PR that removes strings made by Dillon but not merged yet and have a separated files to be uploaded to Crowdin. So this is becoming an actual fork!
I don't think this would be possible to maintain in the future.

Also there is no way we can keep on finding more strings to be removed as most are already reported.

So please make a merge party on language related PR (combined language update + copyright update + Double spaces on text)

**Comment by horus68** [comment]
@Dillon-Brown now that 7.9.1 is released... is it time to get this and related language PR merged?

File: install/language/en_us.lang.php [hunk 2]
```diff
@@ -603,7 +603,13 @@
 	'LBL_OLD_PHP_MSG' => 'The recommended PHP version to install SuiteCRM is %s <br />Although the minimum PHP version required is %s, is not recommended due to the large number of fixed bugs, including security fixes, released in the more modern versions.<br />You are using PHP version %s, which is EOL: <a href="http://php.net/eol.php">http://php.net/eol.php</a>.<br />Please consider upgrading your PHP version. Instructions on <a href="http://php.net/migration70">http://php.net/migration70</a>. ',
 	'LBL_OLD_PHP_OK' => 'I\'m aware of the risks and wish to continue.',
 
-
+    'LBL_DBCONF_TITLE_USER_INFO_LABEL' => 'User',
+    'LBL_DBCONFIG_MSG3_LABEL' => 'Database Name',
+    'LBL_DBCONFIG_MSG3' => 'Name of the database that will contain the data for the SuiteCRM instance you are about to install:',
+    'LBL_DBCONFIG_MSG2_LABEL' => 'Host Name',
+    'LBL_DBCONFIG_MSG2' => 'Name of web server or machine (host) on which the database is located ( such as localhost or www.mydomain.com ):',
+    'LBL_DBCONFIG_B_MSG1_LABEL' => '',
```
Comment (horus68) [inline install/language/en_us.lang.php]: empty string?
`'LBL_DBCONFIG_B_MSG1_LABEL' => '',`

**Comment by horus68** [comment]
@samus-aran you didn't take in consideration my reviews before merging!

**Comment by samus-aran** [comment]
Hi @horus68, am I missing something?  Your reviews was 11 hours ago, and my merge was 22 hours ago, so I believe your reviews were after the merge is that correct?

Is was also a duplication PR opened as well, did that not include the duplication check?

**Comment by likhobory** [comment]
>     LBL_DBCONF_TITLE_USER_INFO_LABEL
>     LBL_DBCONFIG_MSG3_LABEL
>     LBL_DBCONFIG_MSG2_LABEL
>     LBL_DBCONFIG_B_MSG1_LABEL

These are unused language strings - they exist in install/language/en_us.lang.php ONLY.

In any case - good long-awaited work. I'll close all my old issues and open some new with additional unused language strings. 
And I hope that the Spots localization issue #1764 will be resolved too :)

**Comment by daniel-samson** [comment]
@likhobory It turns out is used by the installer. After much testing, we discovered that they are in use. So we have reverted them.

**Comment by horus68** [comment]
@samus-aran something strange with my reviews here... they was made a long time ago as a "Review" but only published 13 hours ago! at least they are now in the "Combined language file update 3"

**Comment by samus-aran** [comment]
Yeah @horus68 We've had that issue before, I think when you make comments but there is like a 'submit' button at the top that you have to confirm to publish all the review comments. Yup, got the guys to pick up your reviews in the additional PR.

**Pull request merged**